docs/wiki/dev-guides/GitHub-Conventions.md

```diff
@@ -34,10 +34,10 @@ docs](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/
 1. Fetch the pull request ref and assign it to a local branch name.
 
    ```bash
-   git fetch upstream pull/<ID>/HEAD/:LOCAL_BRANCH_NAME
+   git fetch <upstream> pull/<ID>/head:LOCAL_BRANCH_NAME
```

Comment:
I believe there is some text further up in this doc that explains making a fork and configuring `upstream` to point to the original repo

Reply:
There isn't

Reply:
its here: https://github.com/Point72/csp/wiki/Release-Process#preparing-and-tagging-a-release

Reply:
One shouldn't have to look at the docs for "Preparing and tagging a release" when trying to check out someone else's PR. They're two unrelated docs pages.

Reply:
Also it's often good to explicitly state what remote you're pushing to/pulling from rather than configuring it. If it's a habit, then it gets rid of the risk of accidentally pushing or pulling from the wrong remote/branch.

Reply:
I agree, though I dont think any suitably experienced maintainer should need to read any of these docs to check out a PR 😉